### PR TITLE
CUR-634 Google classroom integration

### PR DESCRIPTION
**Pull request by CptTony**

### CODE REVIEW

File: src/containers/LMS/GoogleClassroom/Activity/index.js [hunk 1]
```diff
@@ -0,0 +1,96 @@
+import React, { useEffect, useState } from 'react';
+import PropTypes from 'prop-types';
+import { withRouter } from 'react-router-dom';
+
+import { useDispatch } from 'react-redux';
```
Comment (muhammadqamar) [inline src/containers/LMS/GoogleClassroom/Activity/index.js:5]: remove blank space above

File: src/containers/LMS/GoogleClassroom/Activity/index.js [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+              });
+            }
+          }
+        } catch (e) {
+          console.log(e);
+        }
+      });
+
+      const stopXapi = () => clearInterval(checkXapi);
+*/
```
Comment (muhammadqamar) [inline src/containers/LMS/GoogleClassroom/Activity/index.js:80]: is it ok to remove comment out code ?

Comment (CptTony) [reply src/containers/LMS/GoogleClassroom/Activity/index.js:80]: This code is coming back in the next push. Just disabled for now.

File: src/services/gapi.service.js [hunk 3]
```diff
@@ -0,0 +1,22 @@
+import config from 'config';
+import axios from 'axios';
+
+const { apiVersion } = config;
+const gapiBaseUrl = 'https://classroom.googleapis.com/v1';
+
+const getStudentCourses = (token) => {
+  return axios({
```
Comment (muhammadqamar) [inline src/services/gapi.service.js:8]: can we use services/http.service to handle all Axios request?

e.g
const create = (projectId, playlist) => httpService
  .post(`/${apiVersion}/projects/${projectId}/playlists`, playlist)
  .then(({ data }) => data)
  .catch((err) => Promise.reject(err.response.data));

**Review by muhammadqamar — CHANGES_REQUESTED** [review]
Kindly look into the above comments.

File: src/services/gapi.service.js (hunk 3 above)
Comment (CptTony) [reply src/services/gapi.service.js:8]: The http.service is tailored to requests directed at our own API. This service calls on the google classroom API so I made it standalone.

**Review by muhammadqamar — APPROVED** [review]

**Pull request merged**